src/codeu/chat/client/commandline/Chat.java

```diff
@@ -92,6 +93,18 @@ private Panel createRootPanel(final Context context) {
 
     final Panel panel = new Panel();
 
+    panel.register("info", new Panel.Command() {
```

Comment:
I would move this `register` call under the `panel.register("help" ...` call, and in the same order you place it in the System.out.println. Just keeps it consistent and the help being on top gives a nice summary of the other commands registered below.